cli/lib/exec/spawn.js

```diff
@@ -141,9 +141,22 @@ module.exports = {
           stdioOptions.env.DISPLAY = process.env.DISPLAY
         }
 
+        if (process.env.ELECTRON_RUN_AS_NODE === 'true') {
```

Comment:
the Electron patch sets this to `1`, the contract for this env var just seems to be 'anything truthy': https://github.com/electron/electron/blob/87dfe1ba991b7fb82fd734b0af237c95a5bcfa2b/patches/node/refactor_alter_child_process_fork_to_use_execute_script_with.patch

```suggestion
        if (process.env.ELECTRON_RUN_AS_NODE) {
```